README.md

```diff
@@ -17,18 +17,18 @@
   </a>
 </p>
 
-**odoc** is a powerful and flexible documentation generator for OCaml. It reads *doc comments*, demarcated by `(** ... *)`, and transforms them into a variety of output formats, including HTML, LaTeX, and man pages.
+**`odoc`** is a powerful and flexible documentation generator for OCaml. It reads *doc comments*, demarcated by `(** ... *)`, and transforms them into a variety of output formats, including HTML, LaTeX, and man pages.
```

Comment:
`odoc` is the name of the command-line tool but Odoc is the name of the project ? I think the README never mention the command-line tool.

Reply:
I had a lengthy conversation with Jon a few years ago to hammer out the official name for `odoc‘, and I had suggested both ODoc and Odoc, but he wanted the official name to be `odoc‘.